lib/business_time/core_ext/fixnum.rb

```diff
@@ -3,14 +3,21 @@
 #  7.business_days.ago
 #  3.business_days.after(some_date)
 #  4.business_hours.before(some_date_time)
+
 class Fixnum
+  include BusinessTime::Currency
   def business_hours
     BusinessTime::BusinessHours.new(self)
   end
   alias_method :business_hour, :business_hours
 
-  def business_days currency = nil
-    BusinessTime::BusinessDays.new(self, currency)
+  def business_days(*currency)
+    currency = args(*currency)
+    if currency.blank?
+      BusinessTime::BusinessDays.new(self)
+    else
+      BusinessTime::BusinessDays.new(self, *currency)
+    end
```

Comment:
Lines 15-20 are a piece of garbage. Take a look at `BusinessTime::BusinessDays#initialize`, it is ready to receive and correctly deal with an argument of this function as is.

```ruby
def business_days(*currency)
  BusinessTime::BusinessDays.new(self, currency)
end
```